packages/matter.js/src/cluster/interface/OnOffLightingInterface.ts

```diff
@@ -0,0 +1,32 @@
+/**
+ * @license
+ * Copyright 2022-2023 Project CHIP Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/*** THIS FILE IS GENERATED, DO NOT EDIT ***/
+
+import { OnOffLightingCluster, ClusterInterface } from "../index.js";
+import { TypeFromSchema } from "../../tlv/TlvSchema.js";
+
+type OffWithEffectRequest = TypeFromSchema<typeof OnOffLightingCluster.commands.offWithEffect.requestSchema>;
+type OnWithTimedOffRequest = TypeFromSchema<typeof OnOffLightingCluster.commands.onWithTimedOff.requestSchema>;
+
+export interface Common {
+    readonly onOff: boolean;
+    readonly globalSceneControl: boolean;
+    readonly onTime: number | undefined;
+    readonly offWaitTime: number | undefined;
+    readonly startUpOnOff: number | undefined;
+
+    invokeOff(): Promise<void>;
+    invokeOn(): Promise<void>;
+    invokeToggle(): Promise<void>;
+    invokeOffWithEffect(request: OffWithEffectRequest): Promise<void>;
+    invokeOnWithRecallGlobalScene(): Promise<void>;
+    invokeOnWithTimedOff(request: OnWithTimedOffRequest): Promise<void>;
+}
+
+export const OnOffLighting: ClusterInterface<Common, Common> = {
```

Comment:
In fact the "Normal" OnOffDevice is missing here?!

Reply:
Yeah I merged in the ClusterExtend stuff after I started.  It caused problems because I was keying off the cluster ID.  I think I resolved it but we may want some additional metadata...  Right now Pump offers OnOffLighting as an option which I assume is not appropriate.